src/middleware/gateway.service.ts

```
@@ -4,13 +4,14 @@ import { MiddlewareLogger } from 'src/common/loggers/logger.service';
 
 @Injectable()
 export class GatewayService {
-  constructor(private readonly middlewareLogger: MiddlewareLogger) { }
+  constructor(private readonly middlewareLogger: MiddlewareLogger) {}
 
   async handleRequest(
     method: string,
     url: string,
     body: Object,
```

Biome

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)